### PR TITLE
boards/opentitan: Improve README for OT bump

### PR DESCRIPTION
**Pull request by twilfredo**
### Pull Request Overview

This adds extensive build instructions/commands to simplify the process of building OpenTitan and running it with Tock.

Since OT has moved onto a new build system with `bazel` this adds commands/instructions to build only the target we need to run Tock.

### Testing Strategy

Building OpenTitan using these commands and running Tock on Verilator.

Tested with: https://github.com/tock/tock/pull/3056

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [x] Ran `make prepush`.

### CODE REVIEW

**Review by bradjc — DISMISSED** [review]

File: boards/opentitan/README.md [hunk 1]
```diff
@@ -70,56 +70,113 @@ Opentitan is supported on both an FPGA and in Verilator. Slightly different
...
+### Build Boot Rom/OTP Image and FuseSOC
+
+Build **only the targets** we care about. Note: the following commands assume  `bazelisk.sh` has been aliased to `bazel`. You may do so with `alias bazel="PATH_TO/bazelisk.sh"` (from the root of the OpenTitan directory).
 
-Build only the targets we care about.
 ```shell
-./meson_init.sh
-ninja -C build-out sw/device/lib/testing/test_rom/test_rom_export_sim_verilator
-ninja -C build-out sw/device/otp_img/otp_img_sim_verilator.vmem
+# To build the ROM
```
Comment (jrvanwhy) [inline boards/opentitan/README.md:111]: Please call this the "test ROM" -- "ROM" alone refers to the production ROM.

Comment (twilfredo) [reply boards/opentitan/README.md]: Fixed!

**Review by jrvanwhy — APPROVED** [review]

**Comment by twilfredo** [comment]
Ping! Re-based and good to go.

**Review by alistair23 — APPROVED** [review]

**Review by bradjc — APPROVED** [review]

**Pull request merged**